src/main/java/org/openrewrite/java/testing/jmockit/JMockitUtils.java

```diff
@@ -41,7 +41,7 @@ static Optional<JMockitBlockType> getJMockitBlock(Statement s) {
         }
 
         for (JMockitBlockType blockType : JMockitBlockType.values()) {
-            if (TypeUtils.isAssignableTo(blockType.getFqn(), clazz.getType())) {
+            if (TypeUtils.isOfClassType(clazz.getType(), blockType.getFqn())) {
```

Comment:
Why this change to `isOfClassType`?

Reply:
Is because, it was treating NonStrictExpectations as an Expectations because NonStrictExpectations inherit from Expectations. So isAssignable includes inherited types.
https://javadoc.io/doc/org.jmockit/jmockit/1.22/mockit/NonStrictExpectations.html

Let me know if you have a better idea.

Reply:
Got it, just was curious. Although there is a potential disadvantage - if a test uses a class that extends `Expectations`. I imagine this is a rare case though, so we can probably not worry about handling it.